projects/m1/035-the-twelve-days-of-christmas/js/index.js

```diff
@@ -0,0 +1,20 @@
+
+function showMessageSong() {
+const songVerses = [" two turtle doves", "three french hens", 'four calling (colly) birds', "five golden (gold) rings", "six geese a-laying", 'seven swans a-swimming', "eight maids a-milking", "nine ladies dancing", 'ten lords a-leaping', "eleven pipers piping", "twelve drummers drumming"];
+//this variable contain the array with 12 verses of the song.
+const newSongVerses = [];
+//this variable contain an empty array,it will contain the elements generated by the loop.
+for (let i = 0; i < songVerses.length; i++) {
+ //the for loop reads the elements in the array, each time it reads a new element it stops and starts over, printing the elements it reads each time
+  const singleVerse = songVerses[i];
+//this variable will contain the verses of the song read by the loop.
+  newSongVerses.push(singleVerse);
+ //the Push method inserts the various elements read and printed in the loop into the newly created array
+console.log(`On the twelfth day of Christmas, my true love sent to me` + ` ${newSongVerses.reverse()}` + ' and a partridge in a pear tree');
+}//the reverse method allows you to reverse the order of the elements in the array in order to return a correct reading of the console.log request from the exercise
+ 
+}
+console.log('On the twelfth day of Christmas, my true love sent to me a partridge in a pear tree') 
+//print the first verse of the song, only one time.
+showMessageSong(); 
+//call the function to print the song .
```

Comment:
Be careful **not to overdo the comments** that make the **code less readable**.
It's **good practice** to make the **code talk** too _(yours in this case is)_.

Reply:
So, did I use too long comments? Is there a way to figure out what the ideal length is?